transports/noise/src/io/framed.rs

```diff
@@ -18,22 +18,18 @@
 // FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 // DEALINGS IN THE SOFTWARE.
 
-//! This module provides a `Sink` and `Stream` for length-delimited
-//! Noise protocol messages in form of [`NoiseFramed`].
+//! This module provides a `Codec` type implementing the `Encoder` and `Decoder`
+//! traits, making it useful alongside a `asynchronous_codec::Framed` to provide
+//! `Sink` and `Stream` for length-delimited Noise protocol messages.
```

Comment:
Nit: I think it is more idiomatic to have a brief, one-sentence subject line in the docs and then continue on with a longer paragraph.

```suggestion
//! Provides a [`Codec`] type implementing the [`Encoder`] and [`Decoder`] traits.
//!
//! Alongside a [`asynchronous_codec::Framed`] this provides a [`Sink`] and [`Stream`]
//! for length-delimited Noise protocol messages.
```

Reply:
yh, seems fine